app/src/main/java/com/vero/tasky/auth/presentation/registration/RegistrationViewModel.kt

```diff
@@ -49,25 +51,29 @@ class RegistrationViewModel @Inject constructor(
                 updateState(
                     state.copy(
                         name = name,
-                        isNameValid = registrationUseCases.validateNameUseCase(name)
+                        isNameValid = registrationUseCases.validateNameUseCase(name),
+                        isErrorName = false,
+                        nameLabel = R.string.name_hint,
                     )
                 )
             }
             is RegistrationEvent.OnPasswordUpdated -> {
                 updateState(
-                    state.copy(password = event.password)
+                    state.copy(
+                        password = event.password,
+                        isErrorPassword = false,
+                        passwordLabel = R.string.password
+                    )
                 )
             }
             RegistrationEvent.SignUp -> signUp()
         }
     }
 
     private fun signUp() {
-        val isNameValid = isNameValid()
-        val isEmailValid = isEmailValid()
-        val isPasswordValid = isPasswordValid()
+        val validationResult = isPasswordValid()
```

Comment:
`isPasswordValid()` sounds like it returns a boolean. For functions that return a result, I'd rather use a different naming, such as `validatePassword()`